src/splat/util/options.py

```diff
@@ -355,7 +359,19 @@ def _parse_yaml(
     p = OptParser(yaml)
 
     basename = p.parse_opt("basename", str)
-    platform = p.parse_opt_within("platform", str, ["n64", "psx", "ps2", "psp"])
+    is_unsupported_platform = p.parse_optional_opt("is_unsupported_platform", bool)
```

Comment:
I think it would be better to use `parse_opt` here and get rid of the None checks
```suggestion
    is_unsupported_platform = p.parse_opt("is_unsupported_platform", bool, False)
```